snorkel/augmentation/policy/core.py

```diff
@@ -57,11 +94,48 @@ def generate(self) -> List[int]:
         List[int]
             Indices of all TFs in order.
         """
-        return list(range(self._n))
+        return list(range(self.n))
 
 
 class ApplyOnePolicy(ApplyAllPolicy):
     """Apply a single TF to each data point."""
 
-    def __init__(self) -> None:
-        super().__init__(n_tfs=1)
+    def __init__(self, n_per_original: int = 1, keep_original: bool = True) -> None:
+        super().__init__(
+            n_tfs=1, n_per_original=n_per_original, keep_original=keep_original
+        )
+
+
+class ApplyEachPolicy(Policy):
+    """Apply each TF individually to each data point.
+
+    This can be used as a baseline policy when using
+    complex transformations which might degenerate if combined.
+
+    Parameters
+    ----------
+    n_tfs
+        Total number of TFs
+    keep_original
+        Keep untransformed data point in augmented data set? Note that
+        even if in-place modifications are made to the original data
+        point by the TFs being applied, the original data point will
+        remain unchanged.
```

Comment:
Can we give examples? That will help differentiate between the other given policies—a quick glance over the class name is a bit confusing (`ApplyEachPolicy` vs. `ApplyOnePolicy`)